examples/bela/bela_example_analogin_2.scd

```diff
@@ -0,0 +1,20 @@
+Server.default = s = Server("belaServer", NetAddr("192.168.7.2", 57110));
+s.initTree;
+s.startAliveThread;
```

Comment:
… then you don't need `startAliveThread`, and IIRC also no `initTree`.

Reply:
I wonder whether it would be better/more clear to assign remote server to its own variable, instead of overwriting default server, e.g. `~belaServer = Server.remote(...)`. What do you all think?

Reply:
In my (slightly outdated) experience, when you're trying out code with the Bela attached, it's quite intuitive to simply treat the Bela server as the default server and to write/run code using the same idiom that you would do normally. There's not much need to treat it as a special case. So, IMHO I'd definitely set `Server.default` to the bela (as in the existing code), and probably also stick with the existing code use of `s` as the usual handy way of accessing it. (However, I haven't done much SC in the past couple of years!)